images/build/distroless-iptables/Makefile

```diff
@@ -18,10 +18,10 @@ REGISTRY?="gcr.io/k8s-staging-build-image"
 IMAGE=$(REGISTRY)/distroless-iptables
 
 TAG ?= $(shell git describe --tags --always --dirty)
-IMAGE_VERSION ?= v0.1.1
+IMAGE_VERSION ?= v0.1.2
 CONFIG ?= distroless
 BASEIMAGE ?= debian:bullseye-slim
-GORUNNER_VERSION ?= v2.3.1-go1.17.3-bullseye.0
+GORUNNER_VERSION ?= v2.3.1-go1.19.1-bullseye.0
```

Comment:
from where is this image coming?
how can I see the changelog of the gorunner_image?

Reply:
https://github.com/kubernetes/release/tree/master/images/build/go-runner

Reply:
I see, so the only difference is the golang versions used to build it, right?

Reply:
Yes, if I understand correctly.